app/views/ladder/LadderView.coffee

```diff
@@ -41,11 +41,19 @@ module.exports = class LadderView extends RootView
 
   constructor: (options, @levelID, @leagueType, @leagueID) ->
     super(options)
+
+  initialize: ->
     @level = @supermodel.loadModel(new Level(_id: @levelID)).model
+    @levelDescription = marked(@level.get('description')) if @level.get('description')
```

Comment:
This won't work because the level is loaded asynchronously, and so when this line is called the level is not loaded. Something like this would work:

```
@level = @supermodel.loadModel(new Level(_id: @levelID)).model
@level.once 'sync', =>
  @levelDescription = marked(@level.get('description')) if @level.get('description')
```